packages/angular/src/app/components/profile-card/profile-card.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="panel">
+  <div class="title-text panel-header">
+    {{title}} </div>
```

Comment:
Let's change formatting:

```suggestion
  <div class="title-text panel-header">
    {{title}} 
  </div>
```